src/black.ts

```diff
@@ -0,0 +1,209 @@
+const blackVersions: Fig.Generator = {
+  script: "gh release list --repo psf/black",
+  postProcess: (output) => {
+    if (!output.includes("Latest")) {
+      return [];
+    }
+
+    return output.split("\n").map((line) => {
+      return { name: line.split("\t")[0], description: "Version" };
+    });
+  },
+};
+
+const completionSpec: Fig.Spec = {
+  name: "black",
+  description: "Python code formatter",
+  args: {
+    name: "file or directory",
+    template: ["filepaths", "folders"],
```

Comment:
nit: folder is always added as part of `filepaths`.
```suggestion
    template: ["filepaths"],
```